config/webpack.config.js

```diff
@@ -33,7 +44,71 @@ const globalEntryFiles = {
   ],
 };
 
-const configGenerator = (buildOptions, apps) => {
+/**
+ * Get a list of all the entry points.
+ *
+ * If the `entry` CLI argument is passed, only the specified
+ * application entries are built.
+ */
+function getEntryPoints(entry) {
```

Comment:
```suggestion
 * @param {string} [entries] - List of comma delimited entries to build. Builds all entries if no value passed. 
 */
 function getEntryPoints(entries) { 
```
if we're going to document it, might as well use jsdoc syntax. what is the returned value?

Reply:
Good call. I didn't write these (just copied), but I'll add some jsdocs. :+1: